driver-adapters/neon-node-basic/index.test.js

```diff
@@ -0,0 +1,105 @@
+const { handler } = require('./index')
+const { dependencies } = require('./package.json')
+
+jest.setTimeout(30000)
```

Comment:
Is that needed in CI only?

Reply:
And why is it needed?

Reply:
Removed the timeout.

Reply:
Sometimes the test can be slower than 5 seconds default timeout
https://github.com/prisma/ecosystem-tests/actions/runs/6088467323/job/16519182818?pr=3901#step:8:409

I lowered it to 10 seconds.